src/GraphQL.Client/Websocket/GraphQLHttpWebSocket.cs

```diff
@@ -194,10 +194,12 @@ public GraphQLHttpWebSocket(Uri webSocketUri, GraphQLHttpClient client)
                         }
                         catch (Exception e)
                         {
-                            Console.WriteLine(e);
+                            Debug.WriteLine(e);
                             throw;
                         }
 
+                        // TODO: wait for "connection_ack" response
```

Comment:
Technically, the client should be waiting for the `connection_ack` response before proceeding. For now, I put a TODO comment here as I don't have the experience with RX to know how to properly codify this step.

Reply:
In the [Apollo GraphQL over WebSocket Protocol description](https://github.com/apollographql/subscriptions-transport-ws/blob/master/PROTOCOL.md#gql_connection_ack) they write that the server **may** respond with `GQL_CONNECTION_ACK`... so it might be better to not wait for it?

Reply:
Urgh... _may_ is the worst.